build/compile/LzmaArchive.targets

```diff
@@ -115,11 +115,7 @@
       </GenerateNuGetPackagesArchiveVersion>
       <PropertyGroup>
         <NugetPackagesArchiveName>nuGetPackagesArchive.$(NuGetPackagesArchiveVersion).lzma</NugetPackagesArchiveName>
```

Comment:
Unfortunately, the GenerateNuGetPackagesArchiveVersion task takes Stage2Directory as input. And Stage2Directory isn't computed in the pre-build. Unless I move that and what it depends on as well? It depends on $(RepoRoot)